lms/urls.py

```diff
@@ -20,9 +20,7 @@
 from openedx.features.enterprise_support.api import enterprise_enabled
 
 
-# Uncomment the next two lines to enable the admin:
-if settings.DEBUG or settings.FEATURES.get('ENABLE_DJANGO_ADMIN_SITE'):
-    admin.autodiscover()
```

Comment:
That was a bad cherry pick on my part when I was putting this new branch together, should be cleaned up in the commit I just pushed.